plugins/feeds/public/sslblacklist_ja3.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
+from io import StringIO
+from typing import ClassVar
+
+import pandas as pd
+
+from core import taskmanager
+from core.schemas import task
+from core.schemas.observables import ja3
+
+
+class SSLBlacklist_JA3(task.FeedTask):
+    _defaults = {
+        "frequency": timedelta(hours=1),
+        "name": "SSLBlacklist_JA3",
+        "description": "This feed contains JA3 hashes of SSL by AbuseCH",
+    }
+
+    _SOURCE: ClassVar["str"] = "https://sslbl.abuse.ch/blacklist/ja3_fingerprints.csv"
+    _NAMES = ["ja3", "first_seen", "last_seen", "threat"]
+
+    def run(self):
+        response = self._make_request(self._SOURCE, auth=None, verify=True)
+        if response:
+            data = StringIO(response.text)
+
+            df = pd.read_csv(
+                data,
+                delimiter=",",
+                comment="#",
+                names=["ja3_md5", "first_seen", "last_seen", "threat"],
+                parse_dates=["first_seen"],
+            )
+            df = self._filter_observables_by_time(df, "last_seen")
+            df = df.fillna("")
+            for _, row in df.iterrows():
+                self.analyze(row)
+
+    def analyze(self, row):
+        ja3_md5 = row["ja3_md5"]
+        first_seen = row["first_seen"]
+        last_seen = row["last_seen"]
+        threat = row["threat"]
+
+        ja3_obs = ja3.JA3(value=ja3_md5).save()
+
+        context = {}
+        context["first_seen"] = first_seen
+        context["last_seen"] = last_seen
+
+        ja3_obs.add_context(self.name, context)
+
+        if threat:
+            ja3_obs.tag([threat])
```

Comment:
I see that one of the lines has something like 

```
7a29c223fb122ec64d10f0a159e07996,2019-06-09 22:55:29,2020-10-27 09:50:26,None
```

Will `threat` here be `None` in this case? or the STRING None? If it's the latter, then this will tag the observable with `None` which is not desireable.

Reply:
Pandas change None to None python , I've done the test :)

Reply:
![image](https://github.com/yeti-platform/yeti/assets/1478484/0388c07b-b5cc-4540-81c1-b3b812fa6832)